### PR TITLE
remove "Hide layers on map" checkboxInput, and `force_hidden`

### PR DESCRIPTION
**Pull request by DanWismer**
- spatial import (shp) layers, remove hidden, and set visible to FALSE
- remove skip_bm, wrap in try block

### CODE REVIEW

**Comment by DanWismer** [comment]
@edwardsmarc @ricschuster @jeffreyhanson 

I have added branch protection rule to master. It doesn't allow force push, and requires a PR review before merge. However, all organizational owners can bypass branch protection (that includes all of us). 

Our new merging strategy is that `feat` and `fix` branches are to merg into the `dev` branch. And `dev` merges into `master`.  All changes to the `dev` branch can be viewed on https://ncc-dev.carleton.ca/login and all changes done the `master`  is viewed on https://ncc.carleton.ca/login

@edwardsmarc, for reviewing, can you write something like "user tested on dev server" if applicable.  Thanks!

**Comment by DanWismer** [comment]
switching it up again ... code reviews (if needed) will happen on the feat/fix branch PR into dev. 

PR into main doesn't need a review, as all the testing and code review changes will be on dev.  

I have changed the branch protection rules so that a PR is required, but no review necessary. PR required is more of a security check so members that have write access can't just push. 

Open to ideas here, trying to strike the balance and not add too much code mgmt. work

**Review by edwardsmarc — APPROVED** [review]
User tested on dev server with prepped data, and shapefile upload.

**Pull request merged**
